src/main/c/CMakeLists.txt

```
@@ -67,6 +67,14 @@ add_library(seasocks ${SEASOCKS_LIBTYPE}
         util/RootPageHandler.cpp
         $<TARGET_OBJECTS:embedded>
         )
+
+SET_TARGET_PROPERTIES(
```

Just a cosmetic change:
```suggestion
set_target_properties(
```